### DISCUSSION TITLE
Footnotes

### DISCUSSION THREAD

**Discussion by ozgurakgun** [posquit0/Awesome-CV#556] [Q&A]
How does one do footnotes?

I tried the usual \footnote{foo bar} style, but no footnote gets generated.

Thanks!

**Comment by justinbass** [comment]
Not sure what you're trying to do. This works for me (inside document block):

\makecvfooter
  {\today}
  {Name}
  {\thepage}

**Comment by Zinggi** [comment]
I had the same problem as @ozgurakgun.    
Footnotes seem to not work inside a `\cventry ...`.

To work around the issue, I used a `\footnotemark` inside the `\cventry` and then outside the `\cventry` a `\footnotetext{text}`.

This works, but if you need multiple footnotes inside a `\cventry`, it gets a bit more tricky.
In that case I used this trick:
```
\addtocounter{footnote}{-$number of footnotemarks$}
\stepcounter{footnote}\footnotetext{text1}
\stepcounter{footnote}\footnotetext{text2}
```

Hopes this helps others who are in the same situation.

**Comment by fairyofdata** [comment]
I Solved this problem. Thanks @Zinggi
